packages/web-components/src/components.d.ts

```diff
@@ -286,6 +286,24 @@ export namespace Components {
     }
     interface VaFeaturedContent {
     }
+    interface VaFileInput {
+        /**
+          * The text displayed on the button.
+         */
+        "buttontext"?: string;
```

Comment:
@andresriveratoro I can't seem to get these comments to display as the description on the Storybook page (or any default values for that matter). Do you have any insight on how to get that to pull into the page?